torch/_logging/_registrations.py

```diff
@@ -4,7 +4,7 @@
 DYNAMIC = ["torch.fx.experimental.symbolic_shapes", "torch.fx.experimental.sym_node"]
 
 register_log("dynamo", ["torch._dynamo", *DYNAMIC])
-register_log("aot", "torch._functorch.aot_autograd")
+register_log("aot", ["torch._functorch.aot_autograd", "torch._functorch._aot_autograd"])
```

Comment:
Does this need to be `torch._functorch._aot_autograd.jit_compile_runtime_wrappers`? (I'm not sure, but have you confirmed that `TORCH_LOGS="aot"` still does the right thing after this PR?)